capa/features/extractors/script.py

```diff
@@ -0,0 +1,30 @@
+import os
+from typing import Tuple, Iterator
+
+from capa.features.common import OS, OS_ANY, ARCH_ANY, FORMAT_SCRIPT, Arch, Format, Feature, ScriptLanguage
+from capa.features.address import NO_ADDRESS, Address, FileOffsetRangeAddress
+
+LANG_CS = "c_sharp"
```

Comment:
```suggestion
LANG_CS = "C#"
```

Reply:
The problem with this change is that the current language can be used to instantiate the tree_sitter language object, e.g.:

Reply:
Resolved.